deployment/gke-single-cluster/flyte_generated.yaml

```diff
@@ -0,0 +1,1550 @@
+spec:
+  ports:
+  - name: redis
+    port: 6379
+    protocol: TCP
+    targetPort: redis
+  selector:
+    app: redis-resource-manager
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: contour
+  name: contour
+  namespace: heptio-contour
+spec:
+  ports:
+  - nodePort: 30081
+    port: 80
+    protocol: TCP
+  selector:
+    app: contour
+  type: NodePort
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: datacatalog
+  name: datacatalog
+  namespace: flyte
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: datacatalog
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "10254"
+        prometheus.io/scrape: "true"
+      labels:
+        app: datacatalog
+    spec:
+      containers:
+      - command:
+        - datacatalog
+        - --logtostderr
+        - --config
+        - /etc/datacatalog/config/datacatalog_config.yaml
+        - serve
+        image: docker.io/lyft/datacatalog:v0.1.1
+        imagePullPolicy: IfNotPresent
+        name: datacatalog
+        ports:
+        - containerPort: 8088
+        - containerPort: 8089
+        resources:
+          limits:
+            cpu: "0.1"
+            ephemeral-storage: 100Mi
+            memory: 200Mi
+        volumeMounts:
+        - mountPath: /etc/datacatalog/config
+          name: config-volume
+      initContainers:
+      - command:
+        - sh
+        - -c
+        - until pg_isready -h postgres -p 5432; do echo waiting for database; sleep
+          2; done;
+        image: postgres:10.1
+        name: check-db-ready
+        volumeMounts:
+        - mountPath: /etc/datacatalog/config
+          name: config-volume
+      - command:
+        - datacatalog
+        - --logtostderr
+        - --config
+        - /etc/datacatalog/config/datacatalog_config.yaml
+        - migrate
+        - run
+        image: docker.io/lyft/datacatalog:v0.1.1
+        imagePullPolicy: IfNotPresent
+        name: run-migrations
+        volumeMounts:
+        - mountPath: /etc/datacatalog/config
+          name: config-volume
+      serviceAccountName: datacatalog
+      volumes:
+      - emptyDir: {}
+        name: shared-data
+      - configMap:
+          name: datacatalog-config-5g5bbg2k49
+        name: config-volume
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: flyteadmin
+  name: flyteadmin
+  namespace: flyte
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: flyteadmin
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "10254"
+        prometheus.io/scrape: "true"
+      labels:
+        app: flyteadmin
+    spec:
+      containers:
+      - command:
+        - flyteadmin
+        - --logtostderr
+        - --config
+        - /etc/flyte/config/flyteadmin_config.yaml
+        - serve
+        image: docker.io/lyft/flyteadmin:v0.1.4
```

Comment:
this says 0.1.4, but the overlay file has 0.1.1.  I'll remove that at some point.  no reason to specify if it's not being used.